AeroGearOAuth2/OAuth2Module.swift

```diff
@@ -130,15 +130,18 @@ open class OAuth2Module: AuthzModule {
             return
         }
         if let url = URL(string:computedUrl.absoluteString + params) {
-            if self.webView != nil {
-                self.webView!.targetURL = url
-                config.webViewHandler(self.webView!, completionHandler)
-            } else {
+            switch config.webView {
+            case .embeddedWebView:
+                if self.webView != nil {
+                    self.webView!.targetURL = url
+                    config.webViewHandler(self.webView!, completionHandler)
+                }
+            case .externalSafari:
+                UIApplication.shared.openURL(url)
+            case .safariViewController:
                 if #available(iOS 9.0, *) {
```

Comment:
@corinnekrych I was hoping to avoid this change by putting the version check in the enum, but it seems it's needed in both places.